stacks/spica/src/storage/pages/index/index.component.ts

```diff
@@ -48,6 +49,12 @@ export class IndexComponent implements OnInit {
   }
 
   ngOnInit(): void {
+    //NgModelChange doesn't work on Safari browsers
+    let fileElement = this.fileSelector.nativeElement;
+    fileElement.addEventListener("change", () => {
+      this.uploadStorageMany(fileElement.files);
+    });
+
```

Comment:
It should be working on every major browser. 
Can you point out me to the issue so I can take look?  Is there any open issue about this?

Reply:
Actually it should be `//event binding on file input doesn't work on Safari browsers`.
Here is the example;
https://stackoverflow.com/questions/58341044/input-event-on-file-type-not-firing-in-safari-browser

Reply:
ngModelChange works in Safari. See: https://stackblitz.com/edit/angular-mfyfa9?file=src%2Findex.html